main.go

```diff
@@ -48,7 +51,11 @@ import (
 var startTime = time.Now()
 
 func main() {
-	fmt.Println("hanayo " + version.Version)
+	logger := slog.New(slog.NewJSONHandler(os.Stdout, nil))
+
+	slog.SetDefault(logger)
+
+	slog.Info("hanayo " + version.Version)
```

Comment:
```suggestion
	slog.Info("The server is starting up - hanayo v" + version.Version)
```

but furthermore, there should be a way to attach the version as a named variable, so in the resulting log it ends up like:
`{"message": "The server is starting up", "version": "1.2.3"}`

Reply:
best structured logging practice is always to have the `message` be a constant, and have any variables extracted into their own fields